app/src/main/res/xml/preferences.xml

```diff
@@ -1,41 +1,105 @@
 <?xml version="1.0" encoding="utf-8"?>
-<PreferenceScreen xmlns:tools="http://schemas.android.com/tools"
+<PreferenceScreen
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto">
 
     <PreferenceCategory
         app:iconSpaceReserved="false"
-        android:title="@string/settings"
-        android:key="prefgroup"/>
+        android:title="@string/status"
+        android:key="status_category">
 
-    <MultiSelectListPreference
+<!--        <Preference-->
+<!--            app:iconSpaceReserved="true"-->
+<!--            android:enabled="true"-->
+<!--            android:key="permissions_status"-->
+<!--            android:persistent="false"-->
+<!--            />-->
+
+        <Preference
+            app:iconSpaceReserved="true"
+            android:enabled="true"
+            android:key="service_status"
+            android:persistent="false"/>
+
+    </PreferenceCategory>
+
+    <PreferenceCategory
         app:iconSpaceReserved="false"
-    android:enabled="true"
-    android:key="mac"
-    android:persistent="true"
-    android:summary="@string/macTitile"
-    android:title="@string/macDesc" />
-
-    <!--
-    <ListPreference
+        android:title="@string/bluetooth"
+        android:key="bluetooth_category">
+
+        <MultiSelectListPreference
+            app:iconSpaceReserved="false"
+            app:singleLineTitle="false"
+            android:enabled="true"
+            android:key="mac"
```

Comment:
I would like to change this key, but if I do that, people will lose selected BT devices. Since we are bumping version to 3.0, we could do it to keep the code more readable. WDYT?